clang/include/clang/Basic/CodeGenOptions.h

```diff
@@ -72,6 +72,11 @@ class CodeGenOptions : public CodeGenOptionsBase {
     LocalExecTLSModel
   };
 
+  enum TLSDialect {
```

Comment:
Prefer enum class (I think it has been more popular in other parts of Clang, perhaps CodeGen/Sema). `TLSDialect::TraditionalDynamic` and `TLSDialect::TLSDesc`